salsah/src/public/js/jquery.htmleditor.js

```diff
@@ -165,7 +165,8 @@
 							
 						}
 					},
-					toolbar: [ ['Source', 'Bold', 'Italic', 'Underline', 'Strike', 'Subscript', 'Superscript', '-','RemoveFormat', 'Link', 'Unlink'] ] // configuration for toolbar buttons, must conform to `STANDARD_MAPPING`
+					toolbar: [ ['Source', 'Bold', 'Italic', 'Underline', 'Strike', 'Subscript', 'Superscript', '-','RemoveFormat', 'Link', 'Unlink'] ], // configuration for toolbar buttons, must conform to `STANDARD_MAPPING`
+					removeButtons: ''
```

Comment:
I think it comes from Firebug since I use the console. I have the impression it is caused by switching between the source code and the rendered text (toolbar "Source"). The whole editor instance is kept in an iframe which has its own DOM. When switching views, some DOM may be exchanged by another, breaking references. 

I think we should disable source view and merge this pull request.